README.md

```diff
@@ -27,6 +30,24 @@ directory or file to be linted.
 </tr>
 </table>
 
+### Configuration
+
+**Xulieta** tries to find a `xulieta.yaml` file in the root of your project
```

Comment:
nnnnnnnnngggggggghhhhhhh YAML

Is there at least a JSON-Schema somewhere under the organisation?

Reply:
I'm actually using `symfony/config` here. 
`ConfigFileValidation` is validating the schema.

Reply:
`symfony/config` and JSON-Schema are not mutually exclusive: adding a schema makes it possible to auto-complete and pre-validate configs without the need of a runtime.